source/deploy/mobile-app-changelog.md

```diff
@@ -2,12 +2,38 @@
 
 Latest Mattermost Mobile Apps releases:
 
-- [2.7.0 Release](#release)
-- [2.6.0 Release](#id1)
-- [2.5.1 Release](#id6)
-- [2.5.0 Release](#id9)
-- [2.4.0 Release](#id14)
-- [2.3.0 Release](#id19)
+- [2.8.0 Release](#release)
+- [2.7.0 Release](#id1)
+- [2.6.0 Release](#id6)
+- [2.5.1 Release](#id11)
+- [2.5.0 Release](#id14)
+- [2.4.0 Release](#id19)
+
+## 2.8.0 Release
+- Release Date: September 15, 2023
+- Server Versions Supported: Server v7.8.0+ is required. Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device.
+
+### Compatibility
+ - **Upgrade to server version v7.8.0 or later is required.** Support for server [Extended Support Release](https://docs.mattermost.com/upgrade/extended-support-release.html) (ESR) v7.1.0 has ended and upgrading to server ESR v7.8.0 or later is required. As we innovate and offer newer versions of our mobile apps, we maintain backwards compatibility only with supported server versions. Users who upgrade to the newest mobile apps while being connected to an unsupported server version can be exposed to compatibility issues, which can cause crashes or severe bugs that break core functionality of the app.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).	
+ - iPhone 5s devices and later with iOS 12.1+ is required.
+
+### Improvements
+ - Calls: Added incoming call notifications for Direct Message and Group Message calls.
+ - Added some performance improvements.
+ - Removed unneeded requests to the server with malformed mentions.
+ - Adds a snackbar info when the ``Copy Info`` button is clicked on the **About** screen.
```

Comment:
```suggestion
 - Adds snackbar info when the ``Copy Info`` button is tapped on the **About** screen.
```

Reply:
what is snackbar information? (that's a new term for me!)

Reply:
It came from the release note in this PR https://github.com/mattermost/mattermost-mobile/pull/7488. I'll change it to "toast".